bindings/python/iota_sdk/client/client.py

```diff
@@ -502,27 +506,29 @@ def unhealthy_nodes(self) -> List[Dict[str, Any]]:
         """
         return self._call_method('unhealthyNodes')
 
-    def prepare_transaction(self, secret_manager=None, options=None):
+    def prepare_transaction(self, secret_manager: Optional[LedgerNanoSecretManager | MnemonicSecretManager |
+                            SeedSecretManager | StrongholdSecretManager] = None, options=None) -> PreparedTransactionData:
         """Prepare a transaction for signing.
         """
-        return self._call_method('prepareTransaction', {
+        return from_dict(PreparedTransactionData, self._call_method('prepareTransaction', {
```

Comment:
Aren't these supposed to be without `Data` now?

Reply:
Only in the wallet